Modix.Bot/Modules/ReplModule.cs

```diff
@@ -91,16 +82,8 @@ public class ReplModule : ModuleBase
             try
             {
                 var client = _httpClientFactory.CreateClient();
+                res = await client.PostAsync(_replUrl, content);
 
-                using (var tokenSrc = new CancellationTokenSource(30000))
-                {
-                    res = await client.PostAsync(_replUrl, content, tokenSrc.Token);
-                }
-            }
-            catch (TaskCanceledException)
-            {
-                await ModifyOrSendErrorEmbed("Gave up waiting for a response from the REPL service.", message);
-                return;
```

Comment:
Is this safe to remove?

Reply:
Yep, repl gives up after 30 seconds anyway, this check was never actually hit in practice.

Reply:
But does MODiX give up after 30 seconds?

Reply:
it fails with a 502 because the repl process was killed after 30 seconds.

Reply:
![image](https://user-images.githubusercontent.com/4044356/68551401-94e79d80-03c1-11ea-9abf-acf9023d4269.png)